hubspot/plugin.py

```diff
@@ -622,55 +627,55 @@ def hubspot_update_contacts(
 ) -> List[str]:
     """Update multiple contacts in HubSpot.
 
-contact_updates is a dict mapping contact id to a list of tuples with the property names to update, and their new values.
-
-Returns:
-    Contact IDs that have been updated.
-
-The property names MUST be one of the following:
-
-Default Properties of Type String:
-adopter_category: An assessment of whether the contact is likely to use our product.
-company_size: Contact's company size.
-date_of_birth: Contact's date of birth.
-degree: Contact's degree.
-field_of_study: Contact's field of study.
-first_conversion_event_name: First form this contact submitted.
-gender: Contact's gender.
-job_function: Contact's job function.
-jobtitle: A contact's job title.
-hs_lead_status: The contact's sales, prospecting, or outreach status.
-hs_persona: A contact's persona.
-lifecyclestage: The qualification of contacts to sales readiness.
-email: A contact's email address.
-mobilephone: A contact's mobile phone number.
-phone: A contact's primary phone number.
-city: A contact's city of residence.
-state: The contact's state of residence.
-country: The contact's country/region of residence.
-industry: The Industry a contact is in.
-hs_linkedinid: A contact's linkedin id.
-
-Default Properties of Type Number:
-num_associated_deals: Count of deals associated with this contact.
-num_conversion_events: The number of forms this contact has submitted.
-num_unique_conversion_events: The number of different forms this contact has submitted.
-recent_deal_amount: Amount of last closed won deal associated with a contact.
-total_revenue: Sum of all closed-won deal revenue associated with the contact.
-
-Default Properties of Type Datetime:
-first_conversion_date: Date this contact first submitted a form.
-first_deal_created_date: Date first deal was created for contact.
-recent_conversion_date: The date this contact last submitted a form.
-recent_deal_close_date: Date last deal associated with contact was closed-won.
-closedate: Date the contact became a customer.
-
-Default Properties of Type Boolean:
-hs_email_bad_address: The email address associated with this contact is invalid.
-hs_is_contact: Is a contact, has not been deleted and is not a visitor.
-hs_is_unworked: Contact has not been assigned or has not been engaged after last owner assignment/re-assignment.
-hs_sequences_is_enrolled: A yes/no field that indicates whether the contact is currently in a Sequence.
-"""
+    contact_updates is a dict mapping contact id to a list of tuples with the property names to update, and their new values.
```

Comment:
Let's not make this change because these are tokens that the LLM would have to use.

Reply:
Did you observe this?  My belief is that it does not change the content of what's passed to the LLM (the result of [`doc_string()`](https://github.com/d8e-ai/alda/blob/e6463769a510ba4b2d1a2e1d41bf74ca89123b1d/v3/actions/plugin_action.py#L60-L61)), as we [use `get_docstring`](https://github.com/d8e-ai/alda/blob/e6463769a510ba4b2d1a2e1d41bf74ca89123b1d/v3/actions/plugin.py#L540).  [`clean` defaults to `True`](https://docs.python.org/3.11/library/ast.html#ast.get_docstring), so I think all leading whitespace should be removed.

Reply:
Ah ok. I assumed that is was being passed through.